dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/validation/validator/event/DataRelationsValidator.java

```diff
@@ -89,15 +89,15 @@ private void validateProgramStageInProgram(
   }
 
   private void validateRegistrationProgram(
-      Reporter reporter, TrackerBundle bundle, Event event, Program eventProgram) {
-    if (eventProgram.isRegistration()) {
+      Reporter reporter, TrackerBundle bundle, Event event, Program program) {
+    if (program.isRegistration()) {
```

Comment:
since you touched this, can you invert that please and turn it into a guard clause so we get a bit less nesting in here 😅